docs/topics/mta-web-controlling-task-order-with-task-manager.adoc

```diff
@@ -0,0 +1,29 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-08-07
+:_mod-docs-content-type: PROCEDURE
+
+[id="mta-web-controlling-task-order-with-task-manager_{context}"]
+= Controlling the order of task execution
+
+You can use *Task Manager* to preempt a {ProductFullName} task you have scheduled for execution to rearrange execution priority. 
+
+[NOTE]
+====
+You can only set *Preemption* for the following tasks:
+
+* Automated tasks (language discovery or technology discovery)
+* Task you scheduled yourself (application analysis).
+====
+
+.Procedure
+
+. In the {ProductShortName} user interface (UI), select the *Migration* view. 
+. Open the *Task Manager* page.
++
+image::task-manager-page.png[Task Manager]
```

Comment:
I suggest that the image will be somewhere in the beginning of the task manager section:
`docs/topics/mta-web-controlling-tasks-with-task-manager.adoc`